homeassistant/components/matter/config_flow.py

```diff
@@ -222,7 +222,7 @@ async def async_step_manual(
             errors["base"] = "cannot_connect"
         except InvalidServerVersion:
             errors["base"] = "invalid_server_version"
-        except Exception:  # noqa: BLE001
+        except Exception:
             LOGGER.exception("Unexpected exception")
             errors["base"] = "unknown"
```

Comment:
Opened an issue for this one as it should work since we log the exception

https://github.com/astral-sh/ruff/issues/15473